public/static/js/spawned_hosts.js

```diff
@@ -562,6 +577,15 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
       if (action == "attachVolume" && $scope.volumeAttachHost) {
         data.host_id = $scope.volumeAttachHost.id;
       }
+      if (action == "extendVolumeExpiration") {
```

Comment:
Best style is to use === & !== to avoid weird conversions

Reply:
done.
I can't think of a problematic conversion, though.